department-of-veterans-affairs/caseflow
# Shalliburton/13778 status toggle

Resolves [#13328 ](https://github.com/department-of-veterans-affairs/caseflow/issues/13328) and [#13307 ]

### Description
Adds logic to allow deepDiff to toggle request cancelled for virtual hearings

### Acceptance Criteria
- [x] Users can toggle virtual/video hearings without issues

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Video" hearing
1. Click the "Edit hearing details" link
1. Change from a "Video" hearing to a "Virtual" hearing and ensure you get a success message
1. Navigate back to hearings and confirm the hearing is now "Virtual": http://localhost:3000/hearings/schedule
1. Navigate back to the previously selected hearing and change it back to "Video"
1. Once again navigate to hearings and confirm the hearing is back to "Video": http://localhost:3000/hearings/schedule
1. Repeat the steps above transitioning between Virtual and Video at least 2 or 3 times to ensure that it is able to change successfully on each time
